sale_commission/views/account_invoice_view.xml

```diff
@@ -14,6 +14,19 @@
             </field>
         </record>
 
+        <record id="invoice_line_agent_form" model="ir.ui.view">
+            <field name="name">account.invoice.line.agent.form</field>
```

Comment:
It will not be empty in database.
If no name is defined, the name will be autogenerated with model name + type (form / search / ...) 
here for exemple it will be "account.invoice.line.agent" + "form".
Exactly, what you wrote.

(but not a blocking point, it's just a cosmetic comment)